src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3653,22 +3653,15 @@ static_assert(std::size(RatingNames) == 6);
             }
         }
 
-        void MaintenanceDrawBar(DrawPixelInfo& dpi, const ScreenCoordsXY& coords, int32_t value, int32_t colour) const
+        void MaintenanceDrawBar(DrawPixelInfo& dpi, const ScreenCoordsXY& coords, int32_t value, colour_t colour) const
         {
             GfxFillRectInset(dpi, { coords, coords + ScreenCoordsXY{ 149, 8 } }, colours[1], INSET_RECT_F_30);
-            if (colour & kBarBlink)
-            {
-                colour &= ~kBarBlink;
-                if (GameIsNotPaused() && (gCurrentRealTimeTicks & 8))
-                    return;
-            }
```

Comment:
Was this used at all? I remember the 'needs' bars blink in the Guest window, but I don't think I've ever seen them in the Ride window.

Reply:
I cannot remember ever seeing them blink in vanilla, even on very bad values.